contracts/BC_fusion/StakeHub.sol

```diff
@@ -0,0 +1,401 @@
+// SPDX-License-Identifier: GPL-3.0-or-later
+pragma solidity ^0.8.0;
+
+import "@openzeppelin/contracts/proxy/utils/Initializable.sol";
+
+import "./System.sol";
+
+interface IBSCValidatorSet {
+    struct Validator {
+        address consensusAddress;
+        address payable feeAddress;
+        address BBCFeeAddress;
+        uint64 votingPower;
+        // only in state
+        bool jailed;
+        uint256 incoming;
+    }
+}
+
+interface IStakePool {
```

Comment:
Move these interface definition to another files?

Reply:
Sure. I will do that in the end.